libcflib/analyze_pkgs.py

```diff
@@ -13,27 +13,42 @@
 
 
 def file_path_to_import(file_path: str):
+    file_path = file_path.split("site-packages/")[-1].split(".egg/")[-1]
```

Comment:
This doesn't make sure that the `file_path` is in `site-packages` right?

Reply:
This makes sure that `file_path` is in `site-packages` https://github.com/regro/libcflib/pull/55/files#diff-5977cf48a353720d200a6af738fcbb4ab55b308ca0ff907c02b2c77fb191d1fbR35